docs/tex/secciones/01_introduccion.tex

```diff
@@ -0,0 +1,48 @@
+\chapter{Introducción}
+
+A día de hoy, el afán por la lectura está aumentando entre no sólo los jóvenes (el sector que más preocupaba) sino adultos también. Una de las posibles causas de este hecho podría haber sido la pandemia, ya que con la cuarentena muchos han tenido que encontrar nuevos \textit{hobbies} a los que dedicarse desde su hogar para reemplazar actividades al aire libre.
+\\\\
```

Comment:
I'd swear I tried that and didn't work but now that I think about it again it might be because I added it at the end of a paragraph and not in a newline. Will fix, thanks!